load_tests/task_definitions/cloudwatch.json

```diff
@@ -0,0 +1,50 @@
+{
+	"family": "load-test-fluent-bit-cloudwatch-$THROUGHPUT",
+	"taskRoleArn": "$TASK_ROLE_ARN",
+	"executionRoleArn": "$EXECUTION_ROLE_ARN",
+	"containerDefinitions": [
+		{
+			"essential": true,
+			"image": "$FLUENT_BIT_IMAGE",
+			"name": "log_router",
+			"firelensConfiguration": {
+				"type": "fluentbit"
+			},
+			"logConfiguration": {
+				"logDriver": "awslogs",
+				"options": {
+					"awslogs-group": "firelens-container",
+					"awslogs-region": "us-west-2",
+					"awslogs-create-group": "true",
+					"awslogs-stream-prefix": "firelens"
+				}
+			},
+			"cpu": 1024,
+			"memoryReservation": 50
+		 },
+		 {
+			 "essential": true,
+			 "image": "075490442118.dkr.ecr.us-west-2.amazonaws.com/load-test-fluent-bit-logger-in-c:latest",
```

Comment:
should this be hard coded?

Reply:
I hard coded it as these templates are only for load tests. Our validation script only accepts this format of logs. This template doesn't have anything special and if customers don't need to take these if they just want run normal FB.

Reply:
set this as env var.